bin/update-dependencies

```diff
@@ -29,10 +29,30 @@ fi
 git remote set-url "$remote_name" "$repo_uri"
 git push --force-with-lease "$remote_name" "$target_branch"
 
+api_request() {
+	method="$1"
+	shift
+	uri="$1"
```

Comment:
This isn't a URI though. Resource? Subresource? Path? (It's not the whole path but it's _a_ path.)

Reply:
Yeah, I went back and forth on that - `path` is a reasonable suggestion though; fixed.